Source/RapyutaSimulationPlugins/Private/Core/RRActorCommon.cpp

```diff
@@ -205,36 +230,20 @@ void URRActorCommon::OnStartSim()
 
 void URRActorCommon::SetupEnvironment()
 {
-    // Here, we initialize artifacts that utilized play resources
     UWorld* currentWorld = GetWorld();
     checkf(currentWorld, TEXT("[URRActorCommon::SetupEnvironment] Failed fetching Game World"));
 
-    // Fetch Main background environment
-    if (!MainEnvironment)
-    {
-        MainEnvironment = URRUObjectUtils::FindEnvironmentActor(currentWorld);
-        // Not all maps has MainEnvironment setup
-    }
-
-    if (!MainFloor)
-    {
-        MainFloor = URRUObjectUtils::FindFloorActor(currentWorld);
-        // Not all maps has MainFloor setup
-    }
-
-    if (!MainWall)
-    {
-        MainWall = URRUObjectUtils::FindWallActor(currentWorld);
-        // Not all maps has MainWall setup
-    }
+    // By default, these just reference GameState's floor & wall but could be overridable in child class setup
+    SceneFloor = GameState->MainFloor;
```

Comment:
`SceneFloor, SceneWall` are used [here](https://github.com/rapyuta-robotics/RapyutaDataSynth/blob/devel/Source/RapyutaSimDataSynth/Private/SceneCapture/RRCaptureDirector.cpp#L211) in case users want to randomize the background